test/cljs/retabled/core_front.cljs

```diff
@@ -86,6 +86,8 @@
                              }
                             {:valfn    :job
                              :sort     true
+                             :filter   true
```

Comment:
remove `:click-to-filter` as its own keyword, and instead let it be one of the values that can be on `:filter`